Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "d434516416c9015943442988fe91cc7b6c27cd4f3e6a96c867ea00a43b6ff1f6"
```

Comment:
I never used pipenv so there's not much I can check here, I trust you with this code

Reply:
It's pipenv who does this automatically :)

Reply:
Good, otherwise it would be crazy updating this document!